cli/runner/cmd/requests.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+)
+
+//file to contain helper functions for different commands
+
+// Simple struct to hold JSON vals from API call
+type Langs struct {
+	Languages []string `json:"languages"`
+	Error     bool
+}
+
+func getLangListJSON(server string, endpoint string, jsonLangs *Langs) string {
+	resp, err := http.Get(server + endpoint)
+	if err != nil {
+		return "Bad API Call"
```

Comment:
Nit: with the comment for line 18, you can return the `error` type for caller to print (like `return nil, err`). It's kind of small but might help to have an error that says a little more specifically what's wrong like this when the server's not running:

```
Get "http://localhost:8080/api/v1/languages": dial tcp 127.0.0.1:8080: connect: connection refused
```